src/api/operations.ts

```diff
@@ -151,13 +152,16 @@ export const drawLine = (options: {
   thickness: number | PDFNumber;
   color: Color | undefined;
   lineCap?: LineCapStyle;
+  dashArray: (number | PDFNumber)[];
+  dashPhase: number | PDFNumber;
```

Comment:
Adding required options to this interface is actually a breaking change (I know, because I've made the mistake before 😂). So these options need to be marked as options, just like `graphicsState` is. For example:
```ts
  dashArray?: (number | PDFNumber)[];
  dashPhase?: number | PDFNumber;
```
And then we'll need to check if they're present when applying the `setDashPattern` operator:
```ts
setDashPattern(options.dashArray ?? [], options.dashPhase ?? 0)
```